src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -471,6 +471,20 @@ object TezosDatabaseOperations extends LazyLogging {
 
   }
 
+  /** Fetch the latest block level available for each account id stored */
+  def getLevelsForAccounts(ids: Set[AccountId]): DBIO[Seq[(String, BigDecimal)]] =
+    Tables.Accounts
+      .map(table => (table.accountId, table.blockLevel))
+      .filter(_._1 inSet ids.map(_.id))
```

Comment:
Do we not have to put in some max condition? Maybe I am missing something but how do we guarantee we are getting the latest block level for each account?

Reply:
The `accounts` table updates the rows - unlike the accounts_history - so we should only have a single entry per id/level pair.